integration_tests/tests/triggers/time_trigger.rs

```diff
@@ -92,6 +92,7 @@ fn mint_asset_after_3_sec() -> Result<()> {
 }
 
 #[test]
+#[ignore = "should be addressed in #5432"]
```

Comment:
Does this test consistently fail, or is it just _very_ flaky?

If it is flaky but still passes, I would consider extending its retry policy in `nextest.toml`.

Reply:
It appears to be consistently failing, at least in my local environment

Reply:
#5432